wposa-init.php

```diff
@@ -139,6 +147,31 @@
 	);
 
 
+
+	// Field: Separator.
+	$wposa_obj->add_field(
+		'wposa_basic',
+		array(
+			'id'      => 'separator',
+			'type'    => 'separator',
+		)
+	);
+
+
```

Comment:
Can we keep the spacing consistent? I like just one single line of space — if the entire project follows that, it looks good. Thoughts? @deviodigital

Reply:
I agree. :+1: Will get that fixed up in the next round of WordPress Coding Standards updates.